### PR TITLE
[FLAP-116] bootstrap gui testing repo

### PR DESCRIPTION
**Pull request by obonobo**
# Description

- Bootstraps the repo
- Adds selenium and a demo test to show how to work with selenium.
- Adds CI job for running selenium tests.

## Link to work item

Closes FLAP-116

## How to Test

- Install google chrome: <https://www.google.com/intl/en_ca/chrome/>
- Install poetry: <https://github.com/python-poetry/poetry>
- Run: `poetry install`
- Run: `poetry run pytest`

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (obonobo) [inline .vscode/settings.json:1]: @flapflapio/core, not sure if we want to exclude these IDE-related files or not. Are you guys using pycharm or vscode?

**Review by instamine — COMMENTED** [review]
What is the benefit of using poetry instead of pip?

File: .vscode/settings.json (hunk 1 above)
Comment (theNafsters) [reply .vscode/settings.json:1]: We should exclude them. I am going to use pycharm for the selenium tests. Ill ask amneet what he will use

**Review by theNafsters — APPROVED** [review]
Maybe add the setup for mac users

**Pull request merged**